proto/addressmetadata.proto

```diff
@@ -15,6 +39,12 @@ message MetadataField {
     bytes metadata = 2;
 }
 
+// Basic key/value used to store header data.
+message Header {
```

Comment:
Why is this appearing suddenly?

Reply:
Moved it down by accident. I'll put it back above the `MetadataField`

Reply:
It doesn't show that it was moved though?